code/game/objects/structures/misc.dm

```diff
@@ -189,3 +189,123 @@
 
 /obj/structure/computer3frame/laptop
 	name = "laptop frame"
+
+// Dartboard
+#define DOUBLE_BAND 2
+#define TRIPLE_BAND 3
+
+/obj/structure/dartboard
+	name = "dartboard"
+	desc = "A dartboard, loosely secured."
+	icon = 'icons/obj/structures/props/props.dmi'
+	icon_state = "dart_board"
+	density = TRUE
+	unslashable = TRUE
+
+/obj/structure/dartboard/get_examine_text()
+	. = ..()
+	if(contents.len)
```

Comment:
```suggestion
	if(length(contents))
```

Reply:
i know it looks more stupid, but we prefer length() to .len as it's faster (by an absurdly small margin) and won't cause runtimes on null (but contents'll never be null sooo 🤷‍♂️)